openedx/core/djangoapps/catalog/management/commands/cache_programs.py

```diff
@@ -48,33 +143,27 @@ def handle(self, *args, **options):
         except:  # pylint: disable=bare-except
             logger.error('Failed to retrieve program UUIDs.')
             raise
-
-        total = len(uuids)
-        logger.info('Received {total} UUIDs.'.format(total=total))
-
-        programs = {}
+        logger.info('Received {total} UUIDs for site {sitename}'.format(
+            total=len(uuids),
+            sitename=site.domain
+        ))
+        logger.info(str(uuids))
```

Comment:
Please don't do this. A giant block of UUIDs is not helpful. You will log the individual UUIDs later, one at a time as you retrieve their details.